com.gitlab.adnan338.Nixwriter.yaml

```diff
@@ -0,0 +1,47 @@
+id: com.gitlab.adnan338.Nixwriter
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: nixwriter
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=host
+  - --device=all
+  - --system-talk-name=org.freedesktop.UDisks2
+  - --talk-name=org.freedesktop.Notifications
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  # This enables building downloaded crates offline
+  env:
+    CARGO_HOME: /run/build/nixwriter/cargo
+
+modules:
+  - name: nixwriter
+    sources:
+      - type: git
+        url: https://gitlab.com/9898287/nixwriter
+        tag: "3.0.0"
+        commit: "88b28ceb8e0849a98df46d942e20311d862e1512"
+      - deptree.json
+      - type: file
+        path: resources/com.gitlab.adnan338.Nixwriter.desktop
+      - type: file
+        path: resources/com.gitlab.adnan338.Nixwriter.svg
+      - type: file
+        path: resources/com.gitlab.adnan338.Nixwriter.64x64.svg
+      - type: file
+        path: resources/com.gitlab.adnan338.Nixwriter.128x128.svg
+      - type: file
```

Comment:
In general it should be enough to ship only the SVG file in `/app/share/icons/hicolor/scalable/apps`. Resized icons can be dropped.

Reply:
I am shipping them as a flathub requirement (https://github.com/flathub/flathub/wiki/App-Requirements#application-icons)

Reply:
"Application icons should either be included as scalable SVGs or high-resolution PNGs." so one or the another; I will think of improving the wording here.